config.yaml

```diff
@@ -0,0 +1,420 @@
+---
+imports:
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/iam_service_account_v1/iam_service_account_v1.py
+    name: iam_service_account_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/gce_instancetemplate_v1/gce_instancetemplate_v1.py
+    name: gce_instancetemplate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/cloudsql_instance_v1/cloudsql_instance_v1.py
+    name: cloudsql_instance_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/address_v1/address_v1.py
+    name: address_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/constants_v1/constants_v1.py
+    name: constants_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/gce_instance_v2/gce_instance_v2.py
+    name: gce_instance_v2.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/ssl_certificate_v1/ssl_certificate_v1.py
+    name: ssl_certificate_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/httphealthcheck_v1/httphealthcheck_v1.py
+    name: httphealthcheck_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/httploadbalancer_v1/httploadbalancer_v1.py
+    name: httploadbalancer_v1.py
+  - path: https://storage.googleapis.com/rs-gdm-stl-master/backendservice_v1/backendservice_v1.py
+    name: backendservice_v1.py
+  - path: star-binnys-com-ca.pem
+  - path: star-binnys-com-key.pem
+  - path: bucket_v1.py
+  - path: cloudsql_instance_v1.py
+  - path: firewall_rule_v1.py
+
+##############################################################################
+  
+  resources:
+    - type: network_v1
```

Comment:
Invalid type, `network_v1.py` is the correct syntax.  You are not importing the `network_v1.py` template and should be using `compute.v1.network`.

Reply:
Updated